release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1557,7 +1555,34 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
 
 * Previously, the word "Operators" was translated for Chinese users, but the plural translation resulted in a bad user experience. Now the word "Operators" is not translated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945818[*BZ#1945818*])
 
-* Previously, an incorrect code was causing `User` and `Group` details to show unrelated subjects. Now, a code has been added to filter by `User` or `Group`, so the `User` and `Group` details show related subjects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951212[*BZ#1951212*])
+* Previously, an incorrect code was causing the `User` and `Group` details to show unrelated subjects. Now, a code has been added to filter by `User` or `Group`, so the `User` and `Group` details show related subjects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951212[*BZ#1951212*])
+
+* Previously, when you imported a devfile, the OpenShift web console ignored the `build guidance` placeholder container which provided the configuration for environment variables, ports and limits. The new deployment had a second container which could not start because the placeholder image could not be fetched and it missed the required configurations. Now, the `build guidance` container is dropped from the new deployment, and the container adds the environment variables, ports and limit configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[BZ#1952214])
```

Comment:
Add serial comma after 'ports' in first and the last sentence.

Reply:
done